frontend/app/package.json

```diff
@@ -39,6 +39,7 @@
     "lodash": "4.17.21",
     "loglevel": "1.8.0",
     "pinia": "2.0.9",
+    "pinia-plugin-persistedstate": "1.5.1",
```

Comment:
At least from having a look at the available plugins they didn't seem overly complicated to implement, do you think it would be too much effort to replicate the functionality instead of introducing an extra dependency?

Reply:
Okay, I think it's not hard 😆

Reply:
@kelsos updated